### PR TITLE
Fix image cuted by canvas via upload

### PR DESCRIPTION
**Pull request by NaulaN**

### CODE REVIEW

File: src/js/upload-image.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+/////////// Add Image From Directory////////////////
+
+function addImageFromDirectory()
+{
+    for (const file of input.files)
+        if (validFileType(file)) {
+            const image = document.createElement("img");
+            image.src = URL.createObjectURL(file);
+            addPreview(image);
+        } /* TODO do a alert not good file select */
```
Comment (almerion) [inline src/js/upload-image.js:47]: left a toto

File: src/modules/mod_home/cont_home.php [hunk 2]
```diff
@@ -5,14 +5,17 @@
 class ContHome
 {
     private $vue;
-    private $model;
+    public $model;
```
Comment (almerion) [inline src/modules/mod_home/cont_home.php:8]: should stay private 
```suggestion
    private $model;
```

File: src/modules/mod_home/model_home.php [hunk 3]
```diff
@@ -6,12 +6,14 @@
 
 class ModelHome extends Connexion
 {
-    public function fetchImages()
+    public function fetchImages($idcompte = null, $shared = null)
     {
-        $query = "SELECT * FROM photo";
+        if ($idcompte == null || $shared == null)
+            $query = "SELECT * FROM photo";
+        else $query = "SELECT * FROM photo WHERE partager=0 AND idcompte=".$idcompte;
```
Comment (almerion) [inline src/modules/mod_home/model_home.php:13]: if a user isn't connected he has access to all images even those who aren't shared from non admin users ?

**Review by almerion — CHANGES_REQUESTED** [review]

File: src/modules/mod_home/model_home.php (hunk 3 above)
Comment (NaulaN) [reply src/modules/mod_home/model_home.php:13]: If the user is not connected, i will have just all pictures of admin shared or not and nothing into `mes photos`.
But if you want i can put a little condition for not displayed images from admin which not shared.

File: src/js/upload-image.js (hunk 1 above)
Comment (NaulaN) [reply src/js/upload-image.js:47]: bruh ?

File: src/modules/mod_home/model_home.php (hunk 3 above)
Comment (almerion) [reply src/modules/mod_home/model_home.php:13]: nah the problem is that you select all photos and not just those of the admin, if it was just those of the admin it should be somthing like 
select * from photo inner join compte where idrole = 2 
or idrole =1 i don't remember witch is the admin

File: src/js/upload-image.js (hunk 1 above)
Comment (almerion) [reply src/js/upload-image.js:47]: idk some popin for the not good file select or just remove the todo i think

**Pull request merged**